_data/packages.yml

```diff
@@ -1,42 +1,73 @@
+- title: "Monitoring and Alerting"
+  fa_icon: "bar-chart"
+  docs_url: "https://github.com/gruntwork-io/module-aws-monitoring-public"
+  description: |
+    Set up monitoring and alerting using AWS CloudWatch. This Package exposes metrics not available in the AWS
+    hypervisor (e.g. memory and disk space utilization), configures Route 53 health checks for your public endpoints,
+    installs a default set of alerts (e.g. for CPU usage, disk space, 5xx errors) that notify you via email or text
+    message, and aggregates logs from all your applications so you can see them in the CloudWatch console.
+- title: "Docker Cluster"
+  fa_icon: "th"
+  docs_url: "https://github.com/gruntwork-io/module-ecs-public"
+  description: |
+    Deploy your Docker containers on top of the AWS EC2 Container Service (ECS). This Package creates the ECS Cluster,
+    runs your Docker containers as ECS services, supports zero-downtime, rolling deployment of new Docker container
+    versions, configures load balancing, health checks, and service discovery using ELBs, scales the number of nodes
+    and tasks up and down based on traffic, and provides automatic recovery of failed nodes and Tasks.
+- title: "AMI Cluster"
+  fa_icon: "cubes"
+  docs_url: "https://github.com/gruntwork-io/module-asg-public"
+  description: |
+    Deploy your Amazon Machine Images (AMIs) on top of an AWS Auto Scaling Group (ASG). This Package creates the ASG
+    Cluster, supports zero-downtime, rolling deployment of new AMI versions, configures load balancing, health checks,
+    and service discovery using ELBs, scales the number of nodes up and down based on traffic, and provides automatic
+    recovery of failed nodes.
+- title: "Security"
+  fa_icon: "lock"
+  docs_url: "https://github.com/gruntwork-io/module-security-public"
+  description: |
+    Set up best security best practices for managing secrets, credentials, and servers. This Package uses AWS Key
+    Management Service (KMS) to encrypt and decypt secrets so you can securely store them version control, configures
+    servers to accept SSH connections from an authorized list of IAM users that updates automatically, and provides
+    server hardening with automated security updates and tools like fail2ban.
+- title: "Continuous Delivery"
+  fa_icon: "refresh"
+  docs_url: "https://github.com/gruntwork-io/module-ci-public"
+  description: |
+    Set up an automated pipeline to build and deploy code using a CI server such as Jenkins, CircleCI, or Travis. After
+    every commit, this Package build your code, runs your tests, and packages your app into a deployable, versioned
+    artifact (e.g. Docker image or Amazon Machine Image). You can kick off an automated deployment by pushing to a
+    specific branch, creating a new tag, manually running a CI job, or even after every single commit (continuous
+    deployment).
```

Comment:
Updated and fixed: https://github.com/gruntwork-io/gruntwork-io.github.io/commit/3f6bfe327982fb9733f0d6d7c86dd2c0dbe9d46c